packages/core/src/fhir-to-cda/cda-types/shared-types.ts

```diff
@@ -51,7 +52,7 @@ export type CdaAddressUse =
   | "TMP"
   | "WP";
 
-export type CdaGender = "M" | "F" | "UK";
+export type CdaGender = "M" | "F" | "UN";
```

Comment:
so ala all the gender convos we have had, UN = other and UNK = Unknown. I am pretty sure this is the case in CDAs too and not just IHE. Shouldnt this be UNK? Or did you get feedback from the online validator that UNK was invalid?

Reply:
Yes, the online validator was not ok with `UK`, but is good with `UN`. And on the gender mapping function, I posted the link to the CDA documentation that provides acceptable genders. It's listed as `UN` there - [link](https://terminology.hl7.org/5.2.0/ValueSet-v3-AdministrativeGender.html)

Reply:
Ok but then what do we do for patients with gender UNK. We have those in our system now and we need to support them. Can we just not give a gender in those cases or will that throw issues on the validator too?

Reply:
Technically in our system they are U but same point

Reply:
Discussed in slack, and fixed the issue.